browser/src/map/handler/Map.Keyboard.js

```diff
@@ -741,6 +741,13 @@ L.Map.Keyboard = L.Handler.extend({
 			return true;
 		}
 
+		// CTRL + ALT + O for new document. This needs to be handled by the integrator.
```

Comment:
copy paste from ctrl + alt + n? 
not for new document, but rather for opening other document?